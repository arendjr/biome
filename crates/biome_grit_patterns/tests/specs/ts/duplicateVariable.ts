
foo && foo();
foo && bar();
foo && foo.bar();
bar || bar();
