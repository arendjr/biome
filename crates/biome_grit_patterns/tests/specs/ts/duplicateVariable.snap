---
source: crates/biome_grit_patterns/tests/spec_tests.rs
expression: duplicateVariable
---
SnapshotResult {
    messages: [],
    matched_ranges: [
        "2:1-2:13",
    ],
    rewritten_files: [],
    created_files: [],
}
